packages/gatsby-transformer-toml/package.json

```diff
@@ -7,8 +7,7 @@
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
   "dependencies": {
-    "@babel/runtime": "^7.9.6",
```

Comment:
Please revert removal of `@babel/runtime` dep - see https://github.com/gatsbyjs/gatsby/pull/23992#issuecomment-627231557